### PR TITLE
Fix startup chaos

### PR DESCRIPTION
**Pull request by burnstreet**
don't render everything on startup, full screen loading screen

### CODE REVIEW

**Comment by Pseudonian** [comment]
Hello, whenever I try testing this on local, the splash screen for the game shows up for a brief moment and then disappears, and is then stuck on a grey screen. This will need to be fixed before we can approve the PR. Thanks!

**Comment by KhafraDev** [comment]
The ``game`` element's style does not update for either of us. Manually setting its display to "block" fixes this.

**Comment by burnstreet** [comment]
Sorry, I forgot to commit the built dist files. Not used to your workflow yet.
It should work for you now.

File: Synergism.css [hunk 1]
```diff
@@ -7,18 +7,10 @@ html {
...
 }
-body {background-color: #111111 ;
-	transform: scale(0.9);
-	transform-origin: 0 0;
-	overflow-x: hidden;
-	/*overflow-y: hidden;*/
-}
-
 body {
-	color: white;
```
Comment (KhafraDev) [inline Synergism.css:18]: should fix black text issue

**Pull request closed without merging**